debian/jitsi-meet-web-config.postinst

```diff
@@ -182,36 +186,57 @@ case "$1" in
             echo "config.flags.receiveMultipleVideoStreams = true;" >> $JITSI_MEET_CONFIG
         fi
 
-        if [[ "$FORCE_NGINX" = "true" && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
+        NGINX_CONF_PATH="/etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf"
+        OPENRESTY_CONF_PATH="/etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf"
+
+        if [[ "$FORCE_OPENRESTY" = "true" ]]; then
+            NGX_COMMON_CONF_PATH="$OPENRESTY_CONF_PATH"
+            NGX_SVC_NAME=openresty
+            OPENRESTY_NGX_CONF="/usr/local/openresty/nginx/conf/nginx.conf"
+        else
+            NGX_COMMON_CONF_PATH="$NGINX_CONF_PATH"
+            NGX_SVC_NAME=nginx
+        fi
+
+        if [[ ( "$FORCE_NGINX" = "true" || "$FORCE_OPENRESTY" = "true" ) && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
 
             # this is a reconfigure, lets just delete old links
             if [ "$RECONFIGURING" = "true" ] ; then
-                rm -f /etc/nginx/sites-enabled/$JVB_HOSTNAME_OLD.conf
+                rm -f "$NGINX_CONF_PATH"
+                rm -f "$OPENRESTY_CONF_PATH"
                 rm -f /etc/jitsi/meet/$JVB_HOSTNAME_OLD-config.js
             fi
 
             # nginx conf
-            if [ ! -f /etc/nginx/sites-available/$JVB_HOSTNAME.conf ] ; then
-                cp /usr/share/jitsi-meet-web-config/jitsi-meet.example /etc/nginx/sites-available/$JVB_HOSTNAME.conf
-                if [ ! -f /etc/nginx/sites-enabled/$JVB_HOSTNAME.conf ] ; then
-                    ln -s /etc/nginx/sites-available/$JVB_HOSTNAME.conf /etc/nginx/sites-enabled/$JVB_HOSTNAME.conf
+            if [ ! -f "$NGX_COMMON_CONF_PATH" ] ; then
+                cp /usr/share/jitsi-meet-web-config/jitsi-meet.example "$NGX_COMMON_CONF_PATH"
+                if [ ! -f /etc/nginx/sites-enabled/$JVB_HOSTNAME.conf ] && ! [[ "$FORCE_OPENRESTY" = "true" ]] ; then
+                    ln -s "$NGINX_CONF_PATH" /etc/nginx/sites-enabled/$JVB_HOSTNAME.conf
```

Comment:
This is wrong, a link to the same file.

Reply:
Sorry. I don't know how on the Earth this was rolled back to that value. Both of them was intended to have another values.

I've already pushed the fix